roles/kubernetes/node/templates/kubelet-container.j2

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
-/usr/bin/docker run --privileged --rm \
---net=host --pid=host --name=kubelet \
+/usr/bin/docker run --privileged \
+--net=host --pid=host --name=kubelet --restart=on-failure:5 \
```

Comment:
This is an ok approach. We're talking about letting kubelet container restart itself if docker restarts, rather than waiting for systemd to determine that kubelet is running or not (and then ensuing service dependency resolution). I think this covers the gap effectively.